tooling/src/hypothesistooling/installers.py

```diff
@@ -90,9 +93,13 @@ def ensure_ghc():
 def ensure_shellcheck():
     if os.path.exists(SHELLCHECK):
         return
-    update_stack()
-    ensure_ghc()
-    subprocess.check_call([STACK, "install", "ShellCheck"])
+    if shutil.which("apt-get") is not None:
```

Comment:
This change has made it so that the the whole repo tests can only be run as root, which isn't ideal.